gvr-video/app/src/main/java/org/gearvrf/video/VideoMain.java

```diff
@@ -138,9 +86,9 @@ public void run() {
             }
         });
 
-        mRadiosityShader = new RadiosityShader(gvrContext);
-        mAdditiveShader = new AdditiveShader(gvrContext);
-        mScreenShader = new ScreenShader(gvrContext);
+        RadiosityShader mRadiosityShader = new RadiosityShader(gvrContext);
```

Comment:
Since you made these local, please remove the "m" prefix to the variable names. Thanks.